talks/storage-requirements/storage-requirements.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Storage requirements"
+author: "Florian Ziemen"
+subtitle: ""
+format:
+  revealjs:
+    transition: slide
+    slide-number: true
+    code-background: true
+    logo: /static/logos.png
+    footer: "[Talks page](/talks)"
+    theme: [default, ../custom.scss]
+    chalkboard:
+      buttons: true
+    navigation-mode: linear
+    auto-stretch: true
+    center: true
+    mermaid:
+      theme: default
+---
+
+
+# What goes where
+
+* Healpix data for variables requested is duplicated among those centers that have enough space.
```

Comment:
Should we already be picky and differentiate between "duplicated" and "cached"?

Reply:
I guess we won't really be able to have a proper caching infrastructure in place, so it likely will be an active copy process before the hackathon starts.